docs/buildah-build.1.md

```diff
@@ -96,7 +96,31 @@ The value of `[name]` is matched with the following priority order:
 
 **--cache-from**
 
-Images to utilise as potential cache sources. Buildah does not currently support --cache-from so this is a NOOP.
+Repository to utilise as potential cache source. When specified buildah will try to look for
+cached layers in specified repository and will attempt to pull cached layers instead of actually
```

Comment:
The cache logic doesn't read and write layers by themselves, it always reads and writes images.